### PR TITLE
[Bridge] Add bridge

### PR DESCRIPTION
**Pull request by mmyyrroonn**

### CODE REVIEW

File: cstrml/bridge/src/mock.rs [hunk 1]
```diff
@@ -0,0 +1,156 @@
+#![cfg(test)]
```
Comment (badkk) [inline cstrml/bridge/src/mock.rs:4]: Add `ChainSafe` license head?

File: cstrml/bridge_transfer/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,112 @@
+// Ensure we're `no_std` when compiling for Wasm.
```
Comment (badkk) [inline cstrml/bridge_transfer/src/lib.rs]: Change the folder name to `bridge-transfer`

**Review by badkk — APPROVED** [review]

**Comment by badkk** [comment]
Hold on after @BOBOKaa tested

**Comment by MyronFanQiu** [comment]
use the cherry-pick commit to create a new PR

File: cstrml/bridge/src/mock.rs (hunk 1 above)
Comment (mmyyrroonn) [reply cstrml/bridge/src/mock.rs:4]: done

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -814,6 +814,31 @@ impl locks::Config for Runtime {
...
+    type Event = Event;
+    type BridgeCommitteeOrigin = MoreThanHalfCouncil;
+    type Proposal = Call;
+    type BridgeChainId = BridgeChainId;
+    type ProposalLifetime = ProposalLifetime;
+}
+
+parameter_types! {
+    // bridge::derive_resource_id(1, &bridge::hashing::blake2_128(b"CRU"));
+    pub const BridgeTokenId: [u8; 32] = hex_literal::hex!("000000000000000000000000000000608d1bc9a2d146ebc94667c336721b2801");
```
Comment (badkk) [inline runtime/src/lib.rs:835]: Will this be changed?

File: runtime/src/lib.rs [hunk 4]
```diff
@@ -873,6 +898,10 @@ construct_runtime! {
         Sudo: pallet_sudo::{Module, Call, Storage, Config<T>, Event<T>},
         // Multisig module. Late addition.
         Multisig: pallet_multisig::{Module, Call, Storage, Event<T>},
+
+        // ChainBridge
+        ChainBridge: bridge::{Module, Call, Storage, Event<T>} = 100,
```
Comment (badkk) [inline runtime/src/lib.rs]: Delete this

**Review by badkk — CHANGES_REQUESTED** [review]

File: runtime/src/lib.rs (hunk 3 above)
Comment (mmyyrroonn) [reply runtime/src/lib.rs:835]: No. It's a format for the word "CRU"

**Review by badkk — APPROVED** [review]

**Pull request merged**